lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -715,4 +717,35 @@ main .attribute-access-type {
   font-family: var(--font-code);
 }
 
+@media (max-width: 480px) {
+  nav {
+    width: 100%;
+  }
+
+  main {
+    margin: 1em auto;
+    padding: 0 1em;
+    max-width: 100%;
+  }
+
+  #navigation-toggle {
+    top: 10px;
```

Comment:
This rule is overwritten by `#navigation-toggle[aria-expanded="false"]` and becomes redundant.